src/algorithms/PVT/gnuradio_blocks/rtklib_pvt_gs.cc

```diff
@@ -2238,6 +2252,6 @@ int rtklib_pvt_gs::work(int noutput_items, gr_vector_const_void_star& input_item
                         }
                 }
         }
-
+        
```

Comment:
I suspect that **clang-format** would squash this whitespace. Be sure to run it before your final commit.